go/engine/identify2_test.go

```diff
@@ -665,6 +665,33 @@ func TestIdentify2NoSigchain(t *testing.T) {
 	}
 }
 
+// TestIdentifyDeleted tests that identifying a deleted user gives the correct
+// error
+func TestIdentify2Deleted(t *testing.T) {
+	tc := SetupEngineTest(t, "Identify2Deleted")
+	defer tc.Cleanup()
+
+	u := CreateAndSignupFakeUserWithUser(tc, "login")
+	userId := u.User.GetUID()
+	DeleteAccount(tc, u)
+
```

Comment:
Maybe create a new TestContext here and use it for everything below?  So the id happens in a clean home directory?

There could be some weirdness id'ing yourself after a random API call deletes you...